rector.php

```diff
@@ -66,10 +66,7 @@
         __DIR__ . '/bin/validate-phpstan-version.php',
         // tests
         '**/Fixture*',
-        '**/Fixture/*',
```

Comment:
the pattern on the previous line already matches everything the deleted line matches